luarules/gadgets/game_disable_unit_sharing.lua

```diff
@@ -2,10 +2,10 @@ local gadget = gadget ---@type Gadget
 
 function gadget:GetInfo()
 	return {
-		name    = 'Disable Unit Sharing',
-		desc    = 'Disable unit sharing when modoption is enabled',
+		name    = 'Unit Sharing Control',
```

Comment:
If you renamed the gadget name, please rename the gadget filename

Reply:
will do.